asyncapi/json-schemas/pe/dpi/journey_status/dpi-journey-status.md

```diff
@@ -0,0 +1,12 @@
+### DPI Journey Status
+| Field         | Value                                                                                                     |
+|---------------|-----------------------------------------------------------------------------------------------------------|
+| Central Topic | ruter/{operatorId}/{vehicleId}/adt/v4/pe/dpi/journey_status                                               |
+| Schema        | [ dpi-journey-status.json ](json-schemas/pe/dpi/journey_status/dpi-journey-status.json)                   |
+| Producer      | [Ruter DPI](https://github.com/orgs/RuterNo/teams/dpi-team)                                               |
+| Consumer      | PTO, [Ruter DPI](https://github.com/orgs/RuterNo/teams/dpi-team)                                          |
+| Service Level | ✅ External API. Restrictions apply. Only backward compatible changes may happen within the major version. |
+
+The DPI Journey status topic is used to inform the Ruter BO about the correct state (Tilstandsmelding) for DPI.
```

Comment:
please refrase this sentance. The intention of the message is not to inform about the "correct state", better to say "current state" of display or something similar